project/src/main/java/com/google/sps/servlets/CandidateServlet.java

```diff
@@ -98,7 +98,7 @@ private Candidate getCandidateData(String candidateId, String electionName) {
         (String) candidateData.getProperty("party"),
         (String) candidateData.getProperty("email"),
         (String) candidateData.getProperty("phone"),
-        (String) candidateData.getProperty("PhotoUrl"),
+        (String) candidateData.getProperty("photoUrl"),
```

Comment:
I thought @tjyoung99  mentioned the API returned this as "PhotoUrl"?

Reply:
Yes. InfoCompiler currently stores it as "photoUrl" and the backend & frontend uses it as "photoUrl" in the merged version. However, I can change everything to "PhotoUrl" as well

Reply:
Actually, I checked the API: https://developers.google.com/civic-information/docs/v2/elections/voterInfoQuery. The API returns this as "photoUrl"

Reply:
Thanks for confirming

Reply:
No problem!